docs/installation.rst

```diff
@@ -33,17 +33,17 @@ common steps:
 
 .. image:: images/setup1.png
 
--  Extract the downloaded .zip archive. Remember where you extracted it to.
+-  For recent plugin versions (5.5.8 and higher): Note where you the downloaded the omero_ij-5.x.x-all.jar file. Copy that file into the *plugins* folder of Fiji.
```

Comment:
Make indicate to remove the previous versions of the jar if they have it or remove the OMERO.ij folder